### ISSUE TITLE
Image Texture not freed on delete?

### ISSUE THREAD

**Issue by Xeno22** [kivy/kivy#2325]
I load many images from files on my raspberry, at some point, it does not load anything anymore. Also I can't start my app afterwards because of an open gl init error.
I tracked it down to the massive textures in the GPU memory.

I load the image this way:

```
         self.image = Image(source = cachedFile, allow_stretch = fullscreen,
                          keep_ratio = True, mipmap = False,
                         pos = self.pos, size = self.size, keep_data = False,
                         nocache = True
                         )
```

and remove this way:

```
     def remove_oldimage(self, *l):
         print "remove oldimage"
         self.remove_widget(self.oldimage)
         del self.oldimage
         self.oldimage = False

     def removeImage(self, effect, *l):
         if self.image != False:
             self.oldimage = self.image
             self.image = False
             if effect != None:
                 anim = Animation(opacity = 0, t=effect)
                 anim.bind(on_complete = self.remove_oldimage)
                 anim.start(self.oldimage)
             else:
                 self.oldimage.opacity = 0
                 self.remove_oldimage()
```

So my question: did I do something wrong while using the Kivy API, or do I really have to deallocate the texture by myself (and how)?
##

**Comment by derekstavis** [comment]
I have been experiencing the same problem. Seems like that when there's no space left for textures, the program goes pretty crazy, with missing textures blinking left and right, screen managers switching back and forth, etc. Killing and reopening the app deallocates all textures and everything work again.

**Comment by bionoid** [comment]
Maybe this issue was fixed by #5664

**Comment by Julian-O** [comment]
No-one has complained for 5 years. Assuming it was fixed.

If you think this is in error, please let us know.
